src/main/java/com/backendoori/ootw/dto/AvatarAppearanceRequest.java

```diff
@@ -0,0 +1,8 @@
+package com.backendoori.ootw.dto;
+
+public record AvatarAppearanceRequest(
+        String type,
+        boolean sex
```

Comment:
이 부분은 indent 사이즈가 조금 다르네요!

```suggestion
public record AvatarAppearanceRequest(
    String type,
    boolean sex
```